### ISSUE TITLE
Add a way to specify fields to always load with the optimizer

### ISSUE THREAD

**Issue by hiporox** [blb-ventures/strawberry-django-plus#44]
Currently, the optimizer knows to always load the `pk` of an object. It would be nice to be able to specify other fields that need to be loaded every time.

This would be useful because I use a lot of proxy models that load the appropriate class as part of the base model `__init__`. These methods rely on certain fields, and if they are not part of the optimized request, it causes an `N + 1` issue.

I image that the style would read something like:
```
@gql.django.type(MyModel, always=["proxy_model_type"])
class MyModelType:
    ...
```
This would cause the optimizer to always load `proxy_model_type` as part of its optimization

**Comment by bellini666** [comment]
Hey @hiporox ,

Interesting idea... Actually, instead of using `always` we could use the `only`, `select_related` and `prefetch_related` since at the end of the day, what we are going to do is to apply those everytime that type is expected to be returned.

**Comment by hiporox** [comment]
That works for me. I would just want the documentation to be clear that these are being loaded in addition to the other fields

**Comment by bellini666** [comment]
@hiporox this should be fixed in the release 1.11 that I'm doing atm. Please reopen if it is not.
